CMakeLists.txt

```
@@ -170,6 +170,10 @@ file(GLOB InfiniTime_SCREENS
   "${InfiniTime_DIR}/src/displayapp/screens/settings/*.h"
   "${InfiniTime_DIR}/src/displayapp/screens/settings/*.cpp"
 )
+
+# skip files, which currently cause problems when truning on CMake untiy build 
+set_source_files_properties(main.cpp ${LVGL_SOURCES} ${InfiniTime_SCREENS} PROPERTIES SKIP_UNITY_BUILD_INCLUSION ON)
```

just formatting, personal preference of mine

```suggestion
set_source_files_properties(
  main.cpp
  ${LVGL_SOURCES}
  ${InfiniTime_SCREENS}
  PROPERTIES SKIP_UNITY_BUILD_INCLUSION ON)
```